tests/Helpers/data/docComment.php

```diff
@@ -38,8 +38,11 @@
 
 	/**
 	 * Function with doc comment and description
+     * And is multi-line
```

Comment:
Broken indentation.

Reply:
Hmm, must be PhpStorm bug... I used "Convert indend -> To tabs". :/